src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ *  (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ *  (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const values = sourceString.split(';');
+  const stylesObj = {};
+
+  values.filter(value => value.includes(':')).forEach(value => {
+    const key = value.split(':');
+
+    stylesObj[key[0].trim()] = key[1].trim();
```

Comment:
you can add extra variables for `key[0].trim()` and` key[1].trim()` for more readability :)